modules/developing/pages/astra-cli.adoc

```diff
@@ -1,78 +1,26 @@
+:description: {astra_cli} provides a one-stop shop for managing your Astra resources through scripts or commands in your local terminal.
 
-The {company} Astra Command-Line Interface ({astra_cli}) is a set of commands for creating and managing Astra resources.
-{astra_cli} commands are available for {astra_db} and {astra_stream}.
-They're designed to get you working quickly, with an emphasis on automation.
-
-{astra_cli} provides a one-stop shop for managing your Astra resources through scripts or commands in your local terminal.
-The wide variety of capabilities include:
-
-* Creation and management of {astra_db} and {astra_stream} artifacts
-* Querying & data loading
-* Organization and user management
-* Security and token configuration
-
-The advantage for you: {astra_cli} makes it possible to submit commands instead of *or in addition to* using {astra_ui} and {company} API calls.
+The xref:astra-cli:ROOT:index.adoc[{astra_db} Command-Line Interface ({astra_cli})] is a set of commands you can use to create and manage {astra_db} and {product} resources:
 
-{astra_cli} features are provided especially for operators, Site Reliability Engineers (SREs), and developers who want the option of using commands when working with {astra_db} databases and {astra_stream} tenants.
+* Creation and management of {astra_db} databases, {astra_stream} tenants, and their associated artifacts.
+* Querying and data loading.
+* Organization and user management.
+* Security and token configuration.
 
-== Two quick examples
-
-Here are two quick {astra_cli} command examples to demonstrate how Astra CLI manages resources across {astra_db} databases and {astra_stream} tenants from your local terminal.
-
-Let's create an {astra_db} database named `demo` from the command line:
-
-[tabs]
-====
-Astra CLI::
-+
---
-[source,bash,subs="attributes+"]
-----
-astra db create demo -k ks2 --if-not-exist --wait
-----
---
-
-Result::
-+
---
-[source,bash,subs="attributes+"]
-----
-[INFO]  Database 'demo' does not exist. Creating database 'demo' with keyspace 'ks2'
-[INFO]  Database 'demo' and keyspace 'ks2' are being created.
-[INFO]  Database 'demo' has status 'PENDING' waiting to be 'ACTIVE' ...
-[INFO]  Database 'demo' has status 'ACTIVE' (took 103513 millis)
-[OK]    Database 'demo' is ready.
-----
---
-====
-
-Now let's create a Pulsar tenant:
+{astra_cli} commands are available for {astra_db} and {astra_stream}.
+They're designed to get you working quickly, with an emphasis on automation.
 
-[tabs]
-====
-Astra CLI::
-+
---
-[source,bash,subs="attributes+"]
-----
-astra streaming create new-tenant-from-cli
-----
---
+For example, the following command creates a Pulsar tenant:
 
-Result::
-+
---
-[source,bash,subs="attributes+"]
+[source,bash,subs="+quotes"]
 ----
-[OK]    Tenant 'new-tenant-from-cli' has being created.
+astra streaming create **TENANT_NAME** --cloud **CLOUD_PROVIDER** --region **REGION_NAME** --plan **PLAN_TYPE** --namespace **TENANT_INITIAL_NAMESPACE_NAME**
```

Comment:
Define placeholders?

Reply:
This is not really an example they are meant to run, it's just showing that you can use Astra CLI for Streaming stuff. They would need to go to the CLI docs to set up the CLI and find the other Streaming-related commands anyways.